CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+## Changelog ##
```

Comment:
I plan on using the releases as well; we just need a text file for when we convert the README.md -> README.txt for deployment to WP.org. That wp2md script combines the CHANGELOG.md with the README.md file and then converts the whole thing to a WP.org readme. The other benefit is, according the Keep a Changelog standards, we can keep an Unreleased header which we update (and ask contributors to update) with each PR, which then just takes a few minor edits to convert into a release, so we don't have to write up the release each time, we just copy it over.

That said, you did get me thinking about whether it's possible to pull down from the API and convert that to the CHANGELOG, which would be really cool, but we're really close to a 1.0.0, I think, so maybe I'll figure that out and add it in the future.